lib/src/components/NxToast/NxToastContainer.tsx

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useRef} from 'react';
+import FocusContext from './contexts';
+
+import { NxToastContainerProps, nxToastContainerPropTypes } from './types';
+
+export { Props } from './types';
+
+const NxToastContainer = (props: NxToastContainerProps) => {
+  const { children } = props;
+
+  const ref = useRef<HTMLDivElement | null>(null),
+      prevSelectedRef = useRef<Element | null>(null);
+
+  const body = document.getElementsByClassName('nx-body')[0];
```

Comment:
This line might go away as we sort out `adjustFocus`, but for reference this would be better as 
```suggestion
  const body = document.body;
```